docker-compose.yml

```diff
@@ -2,98 +2,168 @@ version: '2'
     depends_on:
-     - config-server
-     - discovery-server
-    entrypoint: ["./dockerize","-wait=tcp://discovery-server:8761","-timeout=60s","--","java", "org.springframework.boot.loader.JarLauncher"]
+      config-server:
+        condition: service_started
+      discovery-server:
+        condition: service_started
+      sql-server:
+        condition: service_started
+    environment:
+      - SPRING_PROFILES_ACTIVE=docker,mysql
+    networks:
+      - default
     ports:
-     - 8082:8082
+      - 8082:8082
 
   vets-service:
-    image: springcommunity/spring-petclinic-vets-service
+    image: docker.io/springcommunity/spring-petclinic-vets-service:2.7.5
     container_name: vets-service
-    mem_limit: 512M
+    mem_limit: 768M
+    restart: on-failure
     depends_on:
-     - config-server
-     - discovery-server
-    entrypoint: ["./dockerize","-wait=tcp://discovery-server:8761","-timeout=60s","--","java", "org.springframework.boot.loader.JarLauncher"]
+      config-server:
+        condition: service_started
+      discovery-server:
+        condition: service_started
+      sql-server:
+        condition: service_started
+    environment:
+      - SPRING_PROFILES_ACTIVE=docker,mysql
+    networks:
+      - default
     ports:
-     - 8083:8083
+      - 8083:8083
 
   api-gateway:
-    image: springcommunity/spring-petclinic-api-gateway
+    image: docker.io/springcommunity/spring-petclinic-api-gateway:2.7.5
     container_name: api-gateway
-    mem_limit: 512M
+    mem_limit: 768M
+    restart: on-failure
     depends_on:
-     - config-server
-     - discovery-server
-    entrypoint: ["./dockerize","-wait=tcp://discovery-server:8761","-timeout=60s","--","java", "org.springframework.boot.loader.JarLauncher"]
-    ports:
-     - 8080:8080
-
-  tracing-server:
-    image: openzipkin/zipkin
-    container_name: tracing-server
-    mem_limit: 512M
+      config-server:
+        condition: service_started
+      discovery-server:
+        condition: service_started
     environment:
-    - JAVA_OPTS=-XX:+UnlockExperimentalVMOptions -Djava.security.egd=file:/dev/./urandom
+      - SPRING_PROFILES_ACTIVE=docker
+    networks:
+      - default
     ports:
-     - 9411:9411
+      - 8080:8080
 
   admin-server:
-    image: springcommunity/spring-petclinic-admin-server
+    image: docker.io/springcommunity/spring-petclinic-admin-server:2.7.5
     container_name: admin-server
-    mem_limit: 512M
+    mem_limit: 768M
+    restart: on-failure
     depends_on:
-     - config-server
-     - discovery-server
-    entrypoint: ["./dockerize","-wait=tcp://discovery-server:8761","-timeout=60s","--","java", "org.springframework.boot.loader.JarLauncher"]
+      config-server:
+        condition: service_started
+      discovery-server:
+        condition: service_started
+    environment:
+      - SPRING_PROFILES_ACTIVE=docker
+    networks:
+      - default
+    ports:
+      - 9090:9090
+
+  ## Third-parties
+
+  sql-server:
```

Comment:
Previous version of the Spring Petclinic started by Docker Compose was using a HSQL database and not a MySQL database.
From your git branch, I've tested the `docker-compose` file  with the `docker-compose up` command.
The whole application didn't start. And I've got this error:
```
vets-service  | 2022-11-29 07:13:50.781 ERROR [vets-service,,] 1 --- [           main] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Exception during pool initialization.
vets-service  |
vets-service  | com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure
vets-service  |
vets-service  | The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
vets-service  |         at com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:174)
```
Does that mean anything to you?